hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -85,28 +93,50 @@ interface IDatanodesState {
   columnOptions: IOption[];
 }
 
-const renderDatanodeStatus = (status: DatanodeStatus) => {
-  const statusIconMap = {
+const renderDatanodeState = (state: DatanodeState) => {
+  const stateIconMap = {
     HEALTHY: <Icon type='check-circle' theme='filled' twoToneColor='#1da57a' className='icon-success'/>,
     STALE: <Icon type='hourglass' theme='filled' className='icon-warning'/>,
-    DEAD: <Icon type='close-circle' theme='filled' className='icon-failure'/>,
-    DECOMMISSIONING: <Icon type='warning' theme='filled' className='icon-warning'/>,
-    DECOMMISSIONED: <Icon type='exclamation-circle' theme='filled' className='icon-failure'/>
+    DEAD: <Icon type='close-circle' theme='filled' className='icon-failure'/>
   };
-  const icon = status in statusIconMap ? statusIconMap[status] : '';
-  return <span>{icon} {status}</span>;
+  const icon = state in stateIconMap ? stateIconMap[state] : '';
+  return <span>{icon} {state}</span>;
+};
+
+const renderDatanodeOpState = (opState: DatanodeOpState) => {
+  const opStateIconMap = {
+    IN_SERVICE: <Icon type='check-circle' theme='filled' twoToneColor='#1da57a' className='icon-success'/>,
+    DECOMMISSIONING: <Icon type='hourglass' theme='filled' className='icon-warning'/>,
+    DECOMMISSIONED: <Icon type='check-circle' theme='filled' className='icon-success'/>,
+    ENTERING_MAINTENANCE: <Icon type='hourglass' theme='filled' className='icon-warning'/>,
+    IN_MAINTENANCE: <Icon type='check-circle' theme='filled' className='icon-success'/>
+  };
```

Comment:
We are showing a green check circle for "DECOMMISSIONED" and "IN_MAINTENANCE" operational states. 

I am not sure if we should differentiate these two from the "IN_SERVICE" state by showing these states in yellow warning signs. Or some other status icon to grab the user's attention to these states maybe?

What do you think?

Reply:
@vivekratnavel Yes, good point.

I'm checking if type `exclamation-circle` works for `DECOMMISSIONED` and `IN_MAINTENANCE` visually.

Do you have any suggestions on that? Any other presets for me to use?

Reply:
You can refer to this document for a list of icons to use - https://ant.design/components/icon/

I would prefer `WarningOutlined` with `icon-warning` class.

Reply:
done. the icon should look like this now: 
<img width="29" alt="Screen Shot 2021-02-23 at 11 20 14 AM" src="https://user-images.githubusercontent.com/50227127/108897252-bf676500-75ca-11eb-87f3-994397b4d60e.png">